### PR TITLE
Add Parse Server Generic Email Adapter to README

### PR DESCRIPTION
**Pull request by premithk**
Added a generic email plugin to send Parse Server password reset and verification emails with generic accounts like Gmail. This module uses the nodemailer library.

### CODE REVIEW

**Comment by addisonElliott** [comment]
What's the status on this PR?

File: README.md [hunk 1]
```diff
@@ -310,7 +310,7 @@ You can also use other email adapters contributed by the community such as:
 - [parse-server-mailgun-adapter-template](https://www.npmjs.com/package/parse-server-mailgun-adapter-template)
 - [parse-server-mailjet-adapter](https://www.npmjs.com/package/parse-server-mailjet-adapter)
 - [simple-parse-smtp-adapter](https://www.npmjs.com/package/simple-parse-smtp-adapter)
-
+- [parse-server-genericemail-adapter](https://www.npmjs.com/package/parse-server-genericemail-adapter)
```
Comment (flovilmart) [inline README.md]: you need an empty line after that please.

**Comment by premithk** [comment]
Have made the changes

File: README.md (hunk 1 above)
Comment (premithk) [reply README.md]: yes, have added the empty line

**Comment by montymxb** [comment]
@premithk The adapter sounds nice, but to be honest I don't feel [this project](https://github.com/premithk/parse-server-genericemail-adapter) is at a point where we should endorse it on our page. It indicates it was last tested against parse-server **2.2.13** and the README could use some work. 

If you wrote this you can check out the other adapters to see a good example of some solid READMEs. Yours doesn't have to copy or anything, but it should be something professional that we would be happy to have as a link to on our page. Additionally, and maybe this can't be changed, but the name is missing a hyphen where I would expect **parse-server-generic-email-adapter**.

Not trying to bash on the adapter though, what it does seems quite useful.

**Comment by premithk** [comment]
@montymxb thank you, i see your point, let me test it against the latest parse-server and update the readme. Regarding the missing hyphen i don't know if i can change that :(

**Comment by premithk** [comment]
@montymxb have updated the readme, can you please check and see if that is enough. (At your leisure). Thank you

**Comment by montymxb** [comment]
@premithk thanks for the changes, there were some additional things I noticed on the repo still. I'll open up a PR there when I get a chance.

**Comment by premithk** [comment]
Sure, thank you! :)

**Comment by montymxb** [comment]
@premithk I'm going to take a look at your repo to suggest some changes. 

On another note regarding the package name, if you're willing to we would like to have the name cleaned up as mentioned above. You could publish a new npm package and deprecate the old one. This has been done before and there's a [deprecation guide for npm modules](https://docs.npmjs.com/cli/deprecate) as well as a little discussion on it in [this SO post](https://stackoverflow.com/questions/28371669/renaming-a-published-npm-module). It's asking a bit, but it would be nice 😄 .

**Comment by premithk** [comment]
@montymxb thank you for the pull request, i've merged those. lemme try and clean up the npm name issue

**Comment by premithk** [comment]
@montymxb i have deprecated the old funky name one and published a new npm package with the above mentioned name https://www.npmjs.com/package/parse-server-generic-email-adapter. Also have updated the readme with the package name changes. Please check at your leisure.

**Comment by premithk** [comment]
@montymxb have updated the readme with the new package names and republished on npm. Thank you.

**Comment by montymxb** [comment]
Awesome! I'm out and about but I'll be back on this later today. I'm going
to give things another good look and test drive the adapter, but I think
this is just about good.

**Comment by premithk** [comment]
@montymxb have merged your changes. thank you

**Comment by montymxb** [comment]
@premithk just rerunning CI to so we can bring this in. Since the requested changes have been made we're definitely good to go.

**Comment by montymxb** [comment]
Thanks again @premithk !

**Comment by premithk** [comment]
Thanks a lot @montymxb

**Pull request merged**